changelog/2.074.0.dd

```diff
@@ -221,13 +221,13 @@ $(P
 It comes with a lot of new features
 $(UL
  $(LI $(LINK2 http://docs.algorithm.dlang.io/latest/mir_ndslice_topology.html, `mir.ndslice.topology`)
-        - Multidimensional `std.range` analog. Includes `bitwise`, `bitpack`, `zip`, `unzip`, `map`, `indexed` and many other features.)
+        \- Multidimensional `std.range` analog. Includes `bitwise`, `bitpack`, `zip`, `unzip`, `map`, `indexed` and many other features.)
```

Comment:
Perhaps the dash should be moved to the end of the previous line to avoid escaping it?